pages/[bip].js

```diff
@@ -48,8 +48,13 @@ export async function getStaticProps({ params }) {
     let data = markdown.toJSON(file)
     if (!data.body) throw new Error(`Failed to parse markdown page for ${bipFile}.md`)
     const title = `BIP${(params.bip)} - ${file.substring(file.indexOf('Title: ') + 7, file.indexOf('Author: '))}`
-    //Replace bip-0001.mediawiki url format by 1
-    const content = data.body.replace(/bip-(\d{1,4}).mediawiki/g, (_, bipNumber) => parseInt(bipNumber))
+    const originalURL = `https://github.com/bitcoin/bips/blob/master/bip-${params.bip.padStart(4, '0')}.mediawiki`

```

Comment:
Maybe you can use `FormatBipAsFile` from utils.js instead doing it directly here.
But you can change the logic of `FormatBipAsFile` by `padStart(4, '0')`. It looks like a way better

Reply:
I like that other approach better. Who knows how well supported is this `padStart` thing.